base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -610,27 +607,14 @@ impl OutputManagerHandle {
         }
     }
 
-    pub async fn get_outputs_by(&mut self, q: OutputBackendQuery) -> Result<Vec<UnblindedOutput>, OutputManagerError> {
```

Comment:
This seems to be a bad merge

Reply:
O, that was a piece of unused code that I cleaned up.